index.bs

```diff
@@ -532,6 +532,14 @@ that agent.  Agents should discard any authentication message whose
 `auth-initation-token` is set and does not match the `at` provided by the
 advertising agent.
 
+In the `psk-min-bits-of-entropy` field of the [=auth-capabilities=] messsage,
+agents may specify the minimum bits of entropy it requires for a PSK, in the
+range of 20 to 60 bits inclusive.  Agents must terminate any connection
+attempting to authenticate with a PSK with fewer than the minimum number of bits
+of entropy and must not attempt to authenticate with an agent that advertises a
+value outside the allowed range (less than 20 or more than 60 bits of
```

Comment:
Nit: the range is already specified, I'd vote for not doubly defining.

Reply:
Removed